src/kvstore.c

```diff
@@ -633,6 +634,37 @@ int kvstoreNumDicts(kvstore *kvs) {
     return kvs->num_dicts;
 }
 
+/* Move dict from one kvstore to another. */
```

Comment:
need thorough review here please, I might have missed something.

Reply:
LGTM, you called `bucketChanged` that fix a bug recently.